src/components/data/DataNavigation.js

```diff
@@ -168,15 +142,14 @@ function FolderSelectorMenu({
     const handleMenuItemClick = (event, index) => {
         setSelectedIndex(index);
         setAnchorEl(null);
-        const storageId = getStorageIdFromPath(router.pathname);
         const relativePath = getRelativePath(
             path,
             userHomePath,
             userTrashPath,
             sharedWithMePath,
             communityDataPath
         );
-        router.push(pathToRoute(storageId, root, relativePath, index));
+        handlePathChange(`${relativePath}${constants.PATH_SEPARATOR}${index}`);
```

Comment:
🤔 Not sure if this will work. It will eval to something like `/analyses/1`

Reply:
Good catch!  I'm not sure how this was working on my machine before I made the PR, but I'll double check.  Either way, this logic doesn't look right 👍

Reply:
These should be fixed now.  I still need to double check the changes in the `utils` about defaulting to `ds` though, will tackle tomorrow.